PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,30 @@
-## High Level Description
+## High-level description
 
-What features does this change enable? What bugs does this change fix? High-Level description of how things changed.
+What features does this change enable? What bugs does this change fix?
 
-## Related Issues
 
-  - [DCOS_OSS-<number>](https://jira.dcos.io/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.
+## Corresponding DC/OS tickets
+
+These DC/OS JIRA ticket(s) must be updated in the moment this PR lands:
```

Comment:
Does this mean that we don't need to update the related tickets?

Reply:
You know just like me that nobody can answer this question because it depends on how the submitter uses their intelligence while filling out the template :-)

The hope is, as I tried to say: When you submit a PR and _you_ want to have a ticket updated by the person that merges your PR then the wording of the new section encourages you to put your ticket into _that_ section instead of into the other one.

Reply:
Hm... I probably want all of my tickets updated when a PR get's merged.

Reply:
> Hm... I probably want all of my tickets updated when a PR get's merged.

Well, in the ideal case this is used for showing the one ticket that must be *closed* in the moment this PR lands. I really hope it's used for that.

However, deviations from the ideal case will come. Because we didn't yet fully think through every little detail. That is also why we cannot really expect from humans to adhere to a super strict set of rules as of now. Because there will be exceptions. For example, as we know there are JIRA tickets that track fixing one issue in multiple branches. In these cases the ticket cannot be *closed* in the moment a PR lands. Which is why I chose "update" instead of "close".

Do you suggest we should reword this?